File: code/349/solution.go
package main /*{{{*/

import (
	"fmt"
	"math"
)

const (
	e4  = int64(10000)
	e5  = int64(100000)
	e6  = int64(1000000)
	e7  = int64(10000000)
	e8  = e4 * e4
	e9  = e8 * 10
	mod = int64(1e9 + 7)
)

func sqrt(a int64) int64 {
	x := math.Sqrt(float64(a))
	return int64(x)
}

/*}}}*/

func main() {
	upper := int64(e9 * e9)
	cnt := (upper - e4) / 104
	ans := cnt * 12
	fmt.Println(ans + 724)
}

File: code/349/solution.cpp
#include "../template/bigint.cpp"
#include "../template/china_reminder.cpp"
#include "../template/euler.cpp"
#include "../template/pell.cpp"
#include "../template/prime.cpp"
#include "../template/prime_cnt.cpp"
#include <iostream>/*{{{*/
#include <map>
#include <math.h>
#include <set>
#include <vector>
const long long e2 = 100;
const long long e3 = e2 * 10;
const long long e4 = e3 * 10;
const long long e5 = e4 * 10;
const long long e6 = e4 * e2;
const long long e7 = e6 * 10;
const long long e8 = e7 * 10;
const long long e9 = e8 * 10;
long long _mod     = e9 + 7;
using namespace std; /*}}}*/

int main() {
    // a(n+104) = a(n) + 12 for n > 9976
    long long upper = e9 * e9;
    long long cnt   = (upper - e4) / 104;
    long long ans   = cnt * 12;
    // BF gets a(10024) = 724
    cout << ans + 724 << endl;
    return 0;
}
